nilearn/image/resampling.py

```diff
@@ -118,11 +118,11 @@ def coord_transform(x, y, z, affine):
     --------
     Transform data from coordinates to brain space. The "affine" matrix
     can be found as the ".affine" attribute of a nifti image, or using
-    the "get_affine()" method for older nibabel installations::
+    the "get_affine()" method for older nibabel installations:
```

Comment:
I think that the double ":" ("::") should be added back.

Reply:
Or maybe this has been done in a different PR, and you are waiting to merge? I may be confused.

Reply:
indeed, these changes were moved (and evolved) in a different PR that just got merged into master. I am now going to merge the current master into this PR, which should solve the issue. Sorry about the confusion, but I was planning to do this and other minor clean-up before requesting for a detailed review.

Reply:
Done.